app/controllers/users_controller.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+# rubocop:todo Style/Documentation
+class UsersController < ActionController::Base
+  before_action :find_user, only: %i[edit update]
+
+  def index
+    @users = User.all
+  end
+
+  def new
+    @user = User.new
+  end
+
+  def edit; end
+
+  def create
+    @user = User.new(user_params)
+    if @user.save
+      redirect_to users_path
+    else
```

Comment:
You should `redirect_to new_user_path` if user is saved ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/forms/user_create.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/forms))